esmvaltool/recipes/ref/recipe_enso_basicclimatology.yml

```
@@ -0,0 +1,179 @@
+# ESMValTool enso basic climatology
+#
+---
+documentation:
+  description: ENSO CLIVAR metrics by Yann Planton, background climatology for REF
+  title: Reproducing ENSO CLIVAR metrics - background climatology
+  authors:
+    - chun_felicity
+    - beucher_romain
+    - planton_yann
+  maintainer:
+    - chun_felicity
+
+datasets:
+  ## one CMIP6 model for REF ##
+  - {dataset: ACCESS-CM2, project: CMIP6, exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 1850, end_year: 2014}
+
+
+
+preprocessors:
+  base_bias: &base_bias
+    custom_order: true
+    # mask_landsea:
+    #   mask_out: land
+    climate_statistics:
+      operator: mean
+      period: full
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_region: &double
+      start_longitude: 210.
+      end_longitude: 270.
+      start_latitude: -15.
+      end_latitude: 15.
+
+  ITCZ_bias:
+    <<: *base_bias
+    convert_units:
+      units: mm/day
+    zonal_statistics:
+      operator: mean
+
+  eq_bias:
+    <<: *base_bias
+    extract_region: &eq_region
+      start_longitude: 150.
+      end_longitude: 270.
+      start_latitude: -5.
+      end_latitude: 5.
+    meridional_statistics:
+      operator: mean
+
+  base_cycle: &base_cycle
+    <<: *base_bias
+    climate_statistics:
+      operator: mean
+      period: monthly
+
+  ITCZ_sea_cycle:
+    <<: *base_cycle
+    convert_units:
+      units: mm/day
+
+  eq_sea_cycle:
+    <<: *base_cycle
+    extract_region:
+      <<: *eq_region
+
+  map_bias_level2: &map_bias
+    custom_order: true
+    # mask_landsea:
+    #   mask_out: land
+    climate_statistics:
+      operator: mean
+      period: full
+    regrid:
+      target_grid: 1x1
+      scheme: linear
+    extract_region: &ext
+      start_longitude: 150.
+      end_longitude: 270.
+      start_latitude: -15.
+      end_latitude: 15.
+
+  map_sea_cyclel2:
+    <<: *map_bias
+    climate_statistics:
+      operator: mean
+      period: monthly
+
+
+diagnostics:
+  diagnostic_metrics:
+    description: run preprocessors on variables for background climatology metrics
+    variables:
+      pr_double:
+        short_name: pr
+        preprocessor: ITCZ_bias
+        mip: Amon
+        additional_datasets:
+          - {dataset: GPCP-SG, project: OBS, type: atmos, tier: 2, mip: Amon, start_year: 1979, end_year: 2018}
+      eq_pr_bias:
+        short_name: pr
+        preprocessor: eq_bias
+        mip: Amon
+        additional_datasets:
+          - {dataset: GPCP-SG, project: OBS, type: atmos, tier: 2, mip: Amon, start_year: 1979, end_year: 2018}
+      eq_sst_bias:
+        short_name: tos
+        preprocessor: eq_bias
+        mip: Omon
+        additional_datasets:
+          - {dataset: TROPFLUX, project: OBS6, type: reanaly, tier: 2, mip: Omon}
```

```suggestion
          - {dataset: TROPFLUX, version: v1, project: OBS6, type: reanaly, tier: 2, start_year: 1979, end_year: 2018}
```
Please specify the `version` facet to avoid the `*` characters in the output filename. This issue will hopefully be solved soon by https://github.com/ESMValGroup/ESMValCore/pull/1943, but until that time you will need to specify the version for datasets that only have it in the filename. Note that there is no need to repeat the `mip` from the variable section since it is the same anyway.